components/patina_smbios/src/manager/global.rs

```diff
@@ -0,0 +1,204 @@
+//! Global state management for SMBIOS manager
```

Comment:
I don't think most of, if any, of these statics are necessary as they can go into your protocol struct, packed behind the protocol itself. You can check out the advanced logger implementation.

These statics should be a last resort. They prevent propper test-ability.